tests/output.rs

```diff
@@ -1193,438 +980,54 @@ fn test_specify_explicit_file_of_unrecognized_file_type_and_language_flag() {
 }
 
 #[test]
-fn test_swift() {
-    assert_sorted_output(
-        "swift_project",
-        r#"
-            $ tree-sitter-grep -q '(value_argument) @c' --language swift
-            example.swift:2:    atPath: "native"
-        "#,
-    );
-}
-
-#[test]
-fn test_swift_auto_language() {
-    assert_sorted_output(
-        "swift_project",
-        r#"
-            $ tree-sitter-grep -q '(value_argument) @c'
-            example.swift:2:    atPath: "native"
-        "#,
-    );
-}
-
-#[test]
-fn test_objective_c() {
-    assert_sorted_output(
-        "objective_c_project",
-        r#"
-            $ tree-sitter-grep -q '(struct_declaration) @c' --language objective-c
-            example.h:4:@property (nonatomic, strong, nullable) NSString *baseURL;
-        "#,
-    );
-}
-
-#[test]
-fn test_objective_c_auto_language() {
-    assert_sorted_output(
-        "objective_c_project",
-        r#"
-            $ tree-sitter-grep -q '(struct_declaration) @c'
-            example.h:4:@property (nonatomic, strong, nullable) NSString *baseURL;
-        "#,
-    );
-}
-
-#[test]
-fn test_objective_c_auto_language_ambiguous_query() {
-    assert_failure_output(
-        "objective_c_project",
-        r#"
-            $ tree-sitter-grep -q '(identifier) @c'
-            File "./example.h" has ambiguous file-type, could be ObjectiveC, C or Cpp. Try passing the --language flag
-        "#,
-    );
-}
-
-#[test]
-fn test_toml() {
+fn test_only_matching() {
```

Comment:
Here are the newly added tests

Reply:
Agreed not good default behavior/"prioritization"

I just ran into this right now when trying to search for/jump to this comment in the Files tab (after having read it in the Conversation tab - that's my workflow because I don't think it lets you aggregate comments as a "review" if you add those comments from the Conversation tab?)